package com.sumith.exception_handling_mcq_sep7;
public class Test27{

public static void main(String [] args){

try{

  //int i;

  return;

}
catch(Exception e)

{

 System.out.println("In catchBlock");

}
finally{

 System.out.println("In Finally");

}
}

}



